api/config/routes.rb

```diff
@@ -187,6 +187,9 @@
         end
       end
 
+      post "projects/:id/ingest", to: "ingests#create"
+      post "texts/:id/ingest", to: "ingests#create"
```

Comment:
Breaking these out, you're gonna end up with routes that look like this instead:

```ruby
namespace :projects do
  resources :ingestions, only: %i[create]
end

namespace :texts do
  resources :ingestions, only: %i[create]
end
```